integration/chaincode/extcc/main.go

```diff
@@ -0,0 +1,263 @@
+/*
```

Comment:
Why do we need yet another copy of the simple chaincode? Can't we just create a main that reuses the logic?

Just adding a subdirectory where we separate the launch from the logic (eg. `integration/chaincode/simple/cmd/{server,client}`) should be sufficient.